src/main/java/racingcar/io/Parser.java

```diff
@@ -0,0 +1,27 @@
+package racingcar.io;
+
+/**
+ * Parses each raw string into appropriate data object.
+ */
+public class Parser {
+    private static final String CAR_NAME_SEPARATOR = ",";
+
+    /**
+     * Parses a raw string indicating names of cars to participate in.
+     * @param line a raw string to parse. Each name of cars is going to be separated by comma(,).
+     * @return given the raw string, an array containing all the names of cars to participate in,
+     *      extracted from the string.
+     */
+    public String[] parseCarNames(String line) {
+        return line.split(CAR_NAME_SEPARATOR);
+    }
+
+    /**
+     * Parses a string indicating an integer into integer data type.
+     * @param integer a string to parse into an integer. integer has to be a string in a form of an integer.
+     * @return an integer from the parameter, integer.
+     */
+    public int parseInt(String integer) {
+        return Integer.parseInt(integer);
+    }
```

Comment:
다른 분들 코드에서 봤는데 이런 메서드 이름은 어떠세요 ? 저는 이게 더 가독성이 좋다고 느꼈어요 !

```suggestion
    public int toInt(String integer) {
        return Integer.parseInt(integer);
    }
```

Reply:
오 되게 좋은 것 같습니다! Integer 클래스의 parseInt 함수의 이름을 그대로 가져다 썼는데, toInt라는 간단한 함수명이 더 나았을 것 같네요.